src/components/Wizard/WizardContainer/WizardContainer.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "./WizardContainer.css";
+import AboutCompany from "./AboutCompany/AboutCompany";
+
+export class WizardContainer extends Component {
+	state = {
+		
+	};
+
+	// onChangeHandler = (e) => {
+	// 	const data = { ...this.state.data };
+	// 	data[e.target.name] = e.target.value;
+	// 	this.setState({ data });
+	// };
+
+	// setWizardProperty = (name,value) => {
+	// 	const data = {...this.state.data}
+	// 	data[name] = value;
+	// 	this.setState({data});
+	// }
+
+	setWizardProperty = (name, value) => {
+		const data = {...this.state}
+		data[name] = value;
+		this.setState(data);
+	}
+
+	setWizardProperties = (properties) => {
+
+		
```

Comment:
What's with the indentation here? Rule of thumb: run checkstyles, reformat, build and then push.